src/tiles/list/tile-list-compact.ts

```diff
@@ -1,20 +1,104 @@
 import { css, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { SortParam } from '@internetarchive/search-service/dist/src/search-params';
 import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
+
+/*
+at 750 creator, title trimmed
+at 530
+
+css elipsis property
+*/
 
 @customElement('tile-list-compact')
 export class TileListCompact extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
   render() {
+    // Todo: Manage different date types
     return html`
-      <h1>${this.model?.title}</h1>
-      <h2>${this.model?.datePublished?.toDateString()}</h2>
+      <div id="list-compact" class="${this.classSize}">
+        <div id="views">
+          ${formatCount(this.model?.viewCount ?? 0, this.formatSize)}
+        </div>
+        <div id="title">
+          <a
+            href="${this.baseNavigationUrl}/details/${this.model?.identifier}"
```

Comment:
We are linking the entire cell inside an `<a>` tag from the `tile-dispatcher`... do you need to link this separately here?

Reply:
Good question. Done that way in tile.inc. Design question. A user might right click on the explicit hyperlink as they aren't aware of the full cell link. We could just style this as a link, but if that, why not just keep it a link. Will ask in offshoot meeting.

Reply:
Ah interesting. Yeah I originally put it at the top so each of the tiles didn't have to do it, but if the list tiles aren't completely linked, we can remove it from the top-level and just wrap the grid tiles so the list tiles can do their own linking.

Reply:
changing based on Jim Shelton in #iaux-collection-browser